pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+SYMPY_VERSION = sm.__version__
+
+
+class TestBodies():
+
+    def test_body_init(self):
+        body = Body('body')
+
+        assert body.name is None
+        assert body.mass == Symbol('mass')
```

Comment:
Yes, every thing is assigned based on the name of the body which provides
us two options when user does not passes a name. One is to make it none and
thus everything else is the default one second is to use assignsymbol
class.  I forgot to change it here but ideally body name should not be none
and mass should be body. Name + '_mass'

On Sat, Jun 27, 2015, 00:13 Tarun Gaba <notifications@github.com> wrote:

> In pydy/tests/test_bodies.py
> <https://github.com/pydy/pydy/pull/249#discussion_r33383712>:
>
> > @@ -0,0 +1,23 @@
> > +#!/usr/bin/env python
> > +
> > +SYMPY_VERSION = sm.__version__
> > +
> > +
> > +class TestBodies():
> > +
> > +    def test_body_init(self):
> > +        body = Body('body')
> > +
> > +        assert body.name is None
> > +        assert body.mass == Symbol('mass')
>
> Does the mass assigned automatically on Body Initialization?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydy/pydy/pull/249/files#r33383712>.
>